### ISSUE TITLE
Problems when adding a package to a project

### ISSUE THREAD

**Issue by e965** [newrelic/newrelic-browser-agent#454]
Hi! I'm trying to add your package to my application, and I ran into some difficulties

1) There are no d.ts declarations for the agent modules, so it was possible to import it only through `// @ts-ignore` comment

2) When trying to import from the package itself (`import { BrowserAgent } from '@newrelic/browser-agent'`), the agent is imported as an interface, not as a class (obviously the [wrong declaration](https://github.com/newrelic/newrelic-browser-agent/blob/40168c71a6538a645237eab7f455113646bd1a97/types.ts#L1))

3) Even if I succeed to import the package, the following error is displayed, and so far I have not been able to progress beyond this step

```
✘ [ERROR] No matching export in "node_modules/@newrelic/browser-agent/dist/esm/common/wrap/wrap-events.js" for import "unwrapEvents"

    node_modules/@newrelic/browser-agent/dist/esm/common/wrap/wrap-xhr.js:10:21:
      10 │ import { wrapEvents, unwrapEvents } from './wrap-events';
         ╵                      ~~~~~~~~~~~~
```

4) I tried to go the workaround and properly redeclare the module, and do the import as described in the second point. But I am still encountering errors.

```
✘ [ERROR] Could not resolve "./custom-agent"

    node_modules/@newrelic/browser-agent/dist/esm/index.js:1:28:
      1 │ export { CustomAgent } from './custom-agent';
        ╵                             ~~~~~~~~~~~~~~~~

✘ [ERROR] Could not resolve "./browser-agent"

    node_modules/@newrelic/browser-agent/dist/esm/index.js:2:29:
      2 │ export { BrowserAgent } from './browser-agent';
        ╵                              ~~~~~~~~~~~~~~~~~

✘ [ERROR] Could not resolve "./worker-agent"

    node_modules/@newrelic/browser-agent/dist/esm/index.js:3:28:
      3 │ export { WorkerAgent } from './worker-agent';
        ╵                             ~~~~~~~~~~~~~~~~
```

Environment - Node 18, Vite 2.2.0, TypeScript 4.9.5 

Codesandbox with reproducabre example - https://codesandbox.io/p/sandbox/thirsty-https-f3s0q5?file=README.md

**Comment by patrickhousley** [comment]
Wow thanks for all the details @e965 . We are excited that you are interesting in our npm package and will look into these issues as soon as possible. Please keep in mind, the npm package is a pre-release and still subject to our current release process. It may take a little time to get an update out there for you. As always, we accept and extremely appreciate feedback as well as pull requests. Thanks for being an early adopter!!!

**Comment by patrickhousley** [comment]
@e965 we have published a new version of the npm package `0.230.0`. Can you please give that a try? Below is a link to an edited version of your reproduction.

https://codesandbox.io/p/sandbox/thirsty-https-f3s0q5?file=README.md

**Comment by e965** [comment]
@patrickhousley Everything seems to be working fine. Thanks for solving the problem!

I will create a new issue if I run into any problems.
